### DISCUSSION TITLE
Restoring Buffer Length Settings in ReVanced Extended patches

### DISCUSSION THREAD

**Discussion by JadenJSJ** [anddea/revanced-patches#433] [Q&A] [answered]
Hi guys!

I've been using the **buffer length modification feature** in ReVanced Extended, which was useful to me for managing YouTube's buffering behavior, especially with my inconsistent internet connection. Unfortunately, this feature was removed due to compatibility issues with newer versions of YouTube, as discussed in [Issue #198](https://github.com/inotia00/ReVanced_Extended/issues/198) in inotia00's repo. The old versions of YouTube Revanced with this patch no longer work so yeah. I was hoping that someone here is willing to make a patch to restore this functionality as it was really useful.

<details close>
<summary>Screenshot of what it looked like previously (click to expand)</summary>

<img src="https://github.com/anddea/revanced-patches/assets/44781632/dd7223ed-790f-4a09-a3c2-d6f868bb769f" alt="Screenshot of what it looked like back then" height="600"/>

</details>

The ability to adjust the following buffer settings was particularly useful: (for those who can't load the screenshot)
- **Maximum Buffer:** The maximum duration of media that the player will attempt to buffer, measured in milliseconds.
*(Default: 120000 ms)*
- **Playback Start:** The required duration of buffered media for playback to commence or resume after user actions like seeking.
*(Default: 2500 ms)*
- **Rebuffer:** The necessary duration of buffered media for playback to continue post-rebuffer, which occurs due to buffer depletion, not user actions.
*(Default: 5000 ms)*

Here's [the commit](https://github.com/inotia00/revanced-patches/commit/96a55c504922316ce8ecf824c2c3f7599294341a) where the feature was removed for reference:
```
commit 96a55c504922316ce8ecf824c2c3f7599294341a
Author: inotia00 <ars258000@gmail.com>
Date:   Mon Jan 30 10:46:54 2023 +0900

    remove `custom-video-buffer` patch https://github.com/inotia00/ReVanced_Extended/issues/198

```

**Comment by anddea** [comment]
Hi. I'll take a look when I have time but I believe it is buggy and almost not working. For now, you can also open an issue in inotia's repo just to remind him about this feature.

**Reply by JadenJSJ** [reply]
Inotia has stopped development of Revanced Extended so I doubt opening an issue would do anything. That is why I opened a discussion here haha

[Discontinuation Announcement](https://github.com/inotia00/revanced-documentation/wiki/Announcement)

**Reply by anddea** [reply, marked as answer]
> Inotia has stopped development of Revanced Extended 

He resumed the development
https://github.com/inotia00/revanced-patches/releases
